All Core Devs Meetings/Meeting 76.md

```diff
@@ -0,0 +1,465 @@
+# Ethereum Core Devs Meeting 76 Notes
+### Meeting Date/Time: Friday 20 September 2019 at 14:00 UTC
```

Comment:
Date looks incorrect

Reply:
Thanks for pointing this out.  It's fixed.